### PR TITLE
Needing to update TLS Version

### PR DESCRIPTION
**Pull request by m2bright**
Needing to update TLS Version for supported SSL Protocol

### CODE REVIEW

File: Main.vb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿
 Module Test
     Public Sub Main()
+        ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11
```
Comment (jlesueur) [inline Main.vb]: https://msdn.microsoft.com/en-us/library/system.net.securityprotocoltype(v=vs.110).aspx says this property can be a bitwise mask. Can we do
```
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12
```

**Review by jlesueur — COMMENTED** [review]
Looks good, one suggestion about including TLS1.2

File: Main.vb (hunk 1 above)
Comment (jlesueur) [reply Main.vb]: sorry, updated the above code to include tls12, instead of referencing tls11 twice :)

**Comment by m2bright** [comment]
@jlesueur Done.

**Comment by m2bright** [comment]
@jlesueur Are you still needing something from me on this?

**Review by jlesueur — APPROVED** [review]

**Comment by jlesueur** [comment]
Looks good, feel free to merge.

**Comment by m2bright** [comment]
@jlesueur I have no permission to merge. You will have to merge it.

**Pull request merged**